host_collection.yml

```diff
@@ -0,0 +1,40 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/host_collection.yml
+    vars:
+      host_collection_state: present
+      expected_change: true
+  - include: tasks/host_collection.yml
+    vars:
+      host_collection_state: present
+      expected_change: false
+  - include: tasks/host_collection.yml
+    vars:
+      host_collection_state: absent
+      expected_change: true
```

Comment:
Can you repeat the absent task with `expected_change: false`, please. We test those modules for idempotence that way.